static/src/javascripts/projects/common/modules/identity/upsell/upsell.js

```diff
@@ -71,37 +29,39 @@ const bindAccountCreation = (el): void => {
     });
 };
 
+type Prefill = {
+    csrfToken: string,
+    accountToken: ?string,
+    email: string,
+    hasPassword: boolean,
+    hasSocialLinks: boolean,
+};
+
+const getPrefill = (el: HTMLElement): Prefill =>
+    // TODO: stricter implementation
```

Comment:
would love to check if dotcom can help us out here – they have a crosswords component in react that may also need data from the server side

Reply:
k, will follow-up with them